hedera-node/src/main/java/com/hedera/services/grpc/marshalling/CustomSchedulesManager.java

```diff
@@ -20,40 +20,37 @@
  * ‍
  */
 
-import com.hedera.services.state.submerkle.EntityId;
-import com.hedera.services.state.submerkle.FcCustomFee;
 import com.hedera.services.store.models.Id;
 import com.hedera.services.txns.customfees.CustomFeeSchedules;
-import org.apache.commons.lang3.tuple.Pair;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class CustomSchedulesManager {
 	private final CustomFeeSchedules customFeeSchedules;
-	private final List<Pair<Id, List<FcCustomFee>>> allManagedSchedules = new ArrayList<>();
+	private final List<CustomFeeMeta> allManagedMeta = new ArrayList<>();
 
 	public CustomSchedulesManager(CustomFeeSchedules customFeeSchedules) {
 		this.customFeeSchedules = customFeeSchedules;
 	}
 
-	public List<FcCustomFee> managedSchedulesFor(EntityId token) {
-		List<FcCustomFee> extantSchedule = null;
-		if (!allManagedSchedules.isEmpty()) {
-			for (var schedule : allManagedSchedules) {
-				if (token.matches(schedule.getKey())) {
-					extantSchedule = schedule.getRight();
+	public CustomFeeMeta managedSchedulesFor(Id token) {
+		CustomFeeMeta extantMeta = null;
+		if (!allManagedMeta.isEmpty()) {
+			for (var meta : allManagedMeta) {
+				if (token.equals(meta.getTokenId())) {
+					extantMeta = meta;
```

Comment:
Will a `break` after the assignment make more sense here?

Reply:
Ah good point, I will do that in PR https://github.com/hashgraph/hedera-services/pull/1834; this one was meant to be just the property change, but Anirudh was enhancing some EETs and merged in that branch.